rest/revocation_test.go

```diff
@@ -1868,6 +1868,82 @@ func TestReplicatorRevocationsWithTombstoneResurrection(t *testing.T) {
 	assert.NoError(t, err)
 }
 
+func TestReplicatorRevocationsWithChannelFilter(t *testing.T) {
+	base.RequireNumTestBuckets(t, 2)
+
+	base.SetUpTestLogging(t, base.LevelDebug, base.KeyAll) // CBG-1981
```

Comment:
```suggestion
```

This is a matter of opinion, but I wouldn't leave any test with full debug logs as a matter of course, this can modified with `SG_TEST_LOG_LEVEL=debug` if you need it debugging your own tests, and if you need to debug a flaky test I might add and remove it. It should keep the size of our jenkins logs down. If you think debug logs would be helpful, I'd probably use a specific key type.